boggle.js

```diff
@@ -1,26 +1,45 @@
 function newBoggleBoard() {
-  return [
-    ['_', '_', '_', '_'],
-    ['_', '_', '_', '_'],
-    ['_', '_', '_', '_'],
-    ['_', '_', '_', '_'],
-  ];
+
+  let boggleBoard = [];
+  for(i=0;i<4;i++)
+  {
+    boggleBoard.push(['_', '_', '_', '_'])
+  }
+  return boggleBoard;
 }
 
+
 /**
  * Prints out a boggle board.
  */
+
+let dice = ['AAEEGN','ELRTTY','AOOTTW','ABBJOO','EHRTVW','CIMOTU','DISTTY','EIOSST','DELRVY','ACHOPS','HIMNQU','EEINSU','EEGHNW','AFFKPS','HLNNRZ','DEILRX'];
```

Comment:
By convention, global constants use ALL CAPS with underscores for spaces, so maybe

```javascript
let BOGGLE_DICE = [...];
```